origocli/commands/events.py

```diff
@@ -86,3 +194,25 @@ def event_stat(self):
             outdata = [hour, day, week]
             out.add_rows(outdata)
             self.print(f"Events for {dataset_id}", out)
+
+    def _resolve_dataset_uri(self):
+        dataset_uri = self.arg("dataset-uri").lower()
+        uri_pattern = r"""
+            ^                           # beginning of string
+            (?:ds:)?                    # non-capturing group, optional "ds:" prefix
+            ([a-z0-9\-]+)               # match one or more characters in range a-z/0-9, and "-" (dataset id)
+            (?:\/([1-9]|[1-9][0-9]+))?  # match digits > 1 prefixed with "/" if present (version)
```

Comment:
Måtte se et par ganger ekstra på denne - så i utgangspunktet ut til å være non-capturing, men det er bare for `/` hvis jeg ikke tar feil?